lib/is_external_link.js

```diff
@@ -13,11 +14,19 @@ const cache = new Cache();
 
 function isExternalLink(input, sitehost, exclude) {
   return cache.apply(`${input}-${sitehost}-${exclude}`, () => {
-    sitehost = parse(sitehost).hostname || sitehost;
+    if (!isAbsoluteUrl(input)) return false;
```

Comment:
`isAbsoluteUrl` matches [any](https://github.com/sindresorhus/is-absolute-url/blob/8a57ccec064d925640860674cd32e7182eeb3596/index.js#L15) protocol. For our use case, `/^http(s)?:\/\//.test(input)` alone should be sufficient.

Reply:
@curbengh Agreed. I will update to it then.

Reply:
Updated to handle `http://`, `https://` and **`//`** prefixed url.